src/DrupalOrg/VersionGroup.php

```diff
@@ -49,7 +49,7 @@ function (string $version) use ($normalizedCurrentVersion): bool {
             }
         );
 
-        $nextVersions = Semver::sort($nextVersions);
+        $nextVersions = Semver::rsort($nextVersions);
```

Comment:
I do not agree with this change.

As I see it we want to act when there is a security update to a project in its current version which prompts us to update to a secure version of the project. Even if we choose not to upgrade right away it should be OK just to update to the next secure version - not necessarily to the latest version of the project.

If a new security update is released subsequently then previous releases may get marked as insecure anyway. Then they will no longer be included in `ProjectFetcher->getSecureVersions()`, no longer included in the version group containing secure versions and only actual secure versions will be considered.

If we accept the current change I fear that we will end up with a JIRA ticket for each new version released after the security update if we choose not to upgrade right away.